tests/integration/steps/integration_steps.go

```diff
@@ -362,12 +451,26 @@ func UninstallK8ssandraHelmRelease(t *testing.T, namespace string) {
 	}
 }
 
+func GetNamespace(name string) (*v1.Namespace, error) {
+	ns := &v1.Namespace{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: name,
+		},
+	}
+	if err := testClient.Get(context.Background(), types.NamespacedName{Namespace: name, Name: name}, ns); err == nil {
+		return ns, nil
+	} else {
+		return nil, err
```

Comment:
Same question here: 
Wouldn't it make the code less verbose if we used an assertion to fail the test here in case err != nil ?
It would remove a conditional branch in the functions that call namespaceIsAbsent().

Is there a good reason for returning the `err` here instead of using an assertion directly or is it to follow Go conventions?

Reply:
Yes, it is more idiomatic to return an error, and I think a function like `GetNamespace` should be a pretty generic helper function.